examples/monai-2D-mednist/README_MonAI_Tutorial.md

```diff
@@ -0,0 +1,12 @@
+
+# Implementing 2D Classification Model with MedNIST Dataset Using FEDn  
+
+This example provides a step-by-step guide to deploying and running a 2D classification model using the MedNIST dataset in a federated environment with the [FEDn framework](https://www.scaleoutsystems.com/framework) developed by [Scaleout Systems](https://www.scaleoutsystems.com/). This example builds on the centralized [example](https://github.com/Project-MONAI/tutorials/blob/main/2d_classification/mednist_tutorial.ipynb) from the MonAI project and adapts it for federated learning settings, utilizing the same code for ease of transition. 
+
+The FEDn framework supports researchers with its robust [open-source SDK](https://fedn.readthedocs.io/en/stable/quickstart.html) and a [public SaaS platform](https://fedn.readthedocs.io/en/stable/studio.html), enabling scalable and efficient federated learning use cases. 
+
+# Getting Started
+
+
+For a step-by-step example guide, click [here](https://github.com/scaleoutsystems/fedn/tree/master/examples/monai-2D-mednist). 
```

Comment:
still the same

Reply:
We have two READMEs: one for the MONAI project repository (README_MonAI_Tutorial.md) and another for the MONAI example in our repository (README) . We are not going to push the code or the detailed README to the MONAI repository. Instead, we will push a short README to their repository with a "click here" link that will direct users to our repository.